lib/core/src/Cardano/Wallet/DB/Store/Transactions/Model.hs

```diff
@@ -0,0 +1,279 @@
+{-# LANGUAGE AllowAmbiguousTypes #-}
```

Comment:
That's a nasty extension — I would like to avoid it as much as possible. I don't even see a specific usage site here — but maybe the type class instances or something. 🤔